packages/graphics/src/Graphics.ts

```diff
@@ -1217,12 +1219,8 @@ export class Graphics extends Container
                 const g = (tintRGB[1] * batchTint[1]) * 255;
                 const b = (tintRGB[2] * batchTint[2]) * 255;
```

Comment:
If you are multiplying three floating point values, then the result need not come out to be an integer. Maybe wrap them around with a `Math.floor/round(...)`. Doing that might be necessary because you're using them in bitwise operations afterwards.